source/create-agg-pipeline.txt

```diff
@@ -376,13 +376,13 @@ Learn More
 .. toctree::
    :titlesonly:
    
-   /agg-pipeline-builder/save-agg-pipeline
-   /agg-pipeline-builder/open-saved-pipeline
-   /agg-pipeline-builder/view-pipeline-explain-plan
-   /agg-pipeline-builder/export-pipeline-to-language
-   /agg-pipeline-builder/create-a-view
-   /agg-pipeline-builder/count-pipeline-results
-   /agg-pipeline-builder/pipeline-custom-collation
-   /agg-pipeline-builder/maxtime-ms-pipeline
-   /agg-pipeline-builder/export-pipeline-results
-   /agg-pipeline-builder/aggregation-pipeline-builder-settings
+   Save Pipeline </agg-pipeline-builder/save-agg-pipeline>
```

Comment:
I think we should leave "Save a Pipeline" and "Open a Pipeline" as-is if we can within the guidelines.

Reply:
Done!